src/OAuth2/Controller/AuthorizeController.php

```diff
@@ -88,6 +99,8 @@ public function __construct(ClientInterface $clientStorage, array $responseTypes
             'require_exact_redirect_uri' => true,
             'redirect_status_code' => 302,
             'enforce_pkce' => false,
+            'enforce_pkce_for_public_clients' => false,
```

Comment:
ℹ️ this should probably default to true for new installs, but that could be a breaking change for existing installs. Leaving this at false allows for releasing as a minor version. Would recommend defaulting to true for the next major version.